features/flags.json

```diff
@@ -26,35 +26,75 @@
       "enabled": true,
       "lastEditedBy": "Andy Haynes",
       "lastEditedAt": "2022-01-31T21:13:09.948Z"
+    },
+    "mainnet_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:06:54.374Z"
+    },
+    "mainnet_STAGING_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:07:28.665Z"
+    },
+    "testnet_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:07:50.110Z"
+    },
+    "testnet_STAGING_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:08:06.806Z"
     }
   },
   "CREATE_USN_CONTRACT": {
     "createdBy": "OlegYak ",
     "createdAt": "2022-03-24T13:11:21.223Z",
     "development": {
-      "enabled": false,
-      "lastEditedBy": "OlegYak ",
-      "lastEditedAt": "2022-03-24T13:11:21.223Z"
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:11:03.325Z"
     },
     "testnet": {
-      "enabled": false,
-      "lastEditedBy": "agileurbanite",
-      "lastEditedAt": "2022-04-26T08:16:00:UTC"
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:11:03.325Z"
     },
     "mainnet": {
-      "enabled": false,
-      "lastEditedBy": "agileurbanite ",
-      "lastEditedAt": "2022-04-26T08:16:00:UTC"
+      "enabled": true,
```

Comment:
I don't think this PR shouldn't be modifying any existing flag states. Since this needs to be landed before we can start using the `*_NEARORG` we run the risk of inadvertently enabling flags in existing environments. Better to enable flags in the current environments once we have all `*.near.org` frontend sites using the `*_NEARORG` environment (and update the configuration repositories so that it can't be automatically reverted to their original values). We just need to be absolutely sure that this feature can't be reenabled as a side effect of this migration.

Reply:
Good point, disabled 👍